ski_conditions/apps/app_scraping/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.2.6 on 2019-11-21 08:57
```

Comment:
Never modify a migration once it has been merged and deployed. There are exceptions to this rule, but for now you should always create new migrations.